lib/rules/html-no-self-closing.js

```diff
@@ -28,6 +28,7 @@ function create (context) {
         return
       }
 
+      // TODO: Check `context.parserServices.getTemplateBodyTokenStore` exists or not.
```

Comment:
This is inside of `registerTemplateBodyVisitor`, so `context.parserServices.getTemplateBodyTokenStore` should always exist.